### ISSUE TITLE
RC Rates do not update after rateprofile switch

### ISSUE THREAD

**Issue by ianrmurphy** [betaflight/blackbox-log-viewer#236]
When switching rate values using rateprofile mapped to an Aux switch, the RC Rates graphs do not update to show the correct values for the new rates.
RC Rates appear to be calculated internal to blackbox-log-viewer (`flightlog.js/calculateSetpointRate()`) from the header data, which is not refreshed upon the rateprofile change. Only an event marker is inserted.
This is likely known behaviour but I could not find any reference to it in previous issues or other documentation.

**Comment by McGiverGim** [comment]
Yes. It is calculated at the beginning ignoring rate changes. Attach a log and I will try to check it when I have time.

**Comment by ianrmurphy** [comment]
Will do, I need to take a log with stock 3.4.0 firmware as I'm currently flying versions related to https://github.com/betaflight/betaflight/pull/6402 that won't show correct behaviour (or reproduce the issue accurately) unless viewed with https://github.com/betaflight/blackbox-log-viewer/pull/237 .

**Comment by ctzsnooze** [comment]
Here's an example, rates were changed twice, we can see the RC Rate calculation not changing after the rate profile change in the log.
[YawTesting.BBL.zip](https://github.com/betaflight/blackbox-log-viewer/files/2347844/YawTesting.BBL.zip)

**Comment by ctzsnooze** [comment]
![rate profile change](https://user-images.githubusercontent.com/11737748/45020325-01566680-b072-11e8-9988-8870c6f8ba5e.jpg)
that's the first rate profile change

**Comment by McGiverGim** [comment]
I reopen this. I had no time to look into it but perhaps...
